### PR TITLE
Add support for passing authorization-requested events to libnotify

### PR DESCRIPTION
**Pull request by xavery**

### CODE REVIEW

File: src/purple-libnotify+.c [hunk 1]
```diff
@@ -169,6 +169,12 @@ _purple_notify_plus_email(PurplePlugin *plugin, const gchar *subject, const gcha
 }
 
 static void
+_purple_notify_auth_request(PurpleAccount *account, const char *remote_user, const char *message, PurplePlugin *plugin)
```
Comment (sardemff7) [inline src/purple-libnotify+.c]: Again, `sender` please.

File: src/purple-libnotify+.c [hunk 2]
```diff
@@ -268,6 +274,11 @@ plugin_load(PurplePlugin *plugin)
 		(PurpleCallback)_purple_notify_plus_email, plugin
 	);
 
+	purple_signal_connect(
+		handle, "authorization-requested", plugin,
```
Comment (sardemff7) [inline src/purple-libnotify+.c]: Of course, change to the `user_’ prefixed name.

File: src/purple-libnotify+.c [hunk 3]
```diff
@@ -169,6 +169,12 @@ _purple_notify_plus_email(PurplePlugin *plugin, const gchar *subject, const gcha
 }
 
 static void
+_purple_notify_auth_request(PurpleAccount *account, const char *remote_user, const char *message, PurplePlugin *plugin)
+{
+	notify_plus_send_name_notification(remote_user, _("Authorization request from %s"), message, NULL, NULL);
```
Comment (sardemff7) [inline src/purple-libnotify+.c]: I want to keep the buddy name as the first thing, to quickly have the context. More like “%s requested authorization”.
(Seems I failed for some other events, let’s not continue.)

**Comment by xavery** [comment]
Thanks for the review, changes pushed.

**Comment by sardemff7** [comment]
As an idea for a further improvement if you feel like it, maybe we could add actions to the notification to directly accept/reject the request?

**Comment by sardemff7** [comment]
And thanks again for this code. :-)

**Comment by xavery** [comment]
Adding accept/reject buttons is a great idea! I'll try to cook something up.

**Pull request merged**